src/attestedclaim/AttestedClaim.ts

```diff
@@ -43,24 +77,75 @@ export default class AttestedClaim implements IAttestedClaim {
     this.attestation = Attestation.fromObject(attestation)
   }
 
+  /**
+   * @description (ASYNC) Verifies whether this attested claim is valid. It is valid if:
+   * * the data is valid (see [[verifyData]]);
+   * and
+   * * the [[Attestation]] object associated to this attestated claim is valid (see [[Attestation.verify]], where the **chain** is queried).
+   *
+   * Upon presentation of an attested claim, a verifier would call this [[verify]] function.
+   * @returns A promise containing whether this attested claim is valid.
+   * @example
+   * ```javascript
+   * attestedClaim.verify().then(data => {
+   *    console.log('isVerified', data);
+   * });
+   * ```
+   */
   public async verify(): Promise<boolean> {
     if (!this.verifyData()) {
       Promise.resolve(false)
     }
     return this.attestation.verify()
   }
 
+  /**
+   * @description Verifies whether the data of this attested claim is valid. It is valid if:
+   * * the [[RequestForAttestation]] object associated with this attested claim has valid data (see [[RequestForAttestation.verifyData]]);
+   * and
+   * * the hash of the [[RequestForAttestation]] object associated to this attested claim, and the hash of the [[Claim]] associated to this attestated claim are the same.
+   * @returns Whether the attestated claim's data is valid.
+   * @example
+   * ```javascript
+   * const isDataValid = attestedClaim.verifyData();
+   * ```
+   */
   public verifyData(): boolean {
     return (
       this.request.verifyData() &&
       this.request.hash === this.attestation.claimHash
     )
   }
 
+  /**
+   * @description Gets the hash of the claim that corresponds to this attestation.
+   * @returns claimHash The hash of the claim that corresponds to this attestation.
+   * @example
+   * ```javascript
+   * const claimHash = attestation.getHash();
+   * ```
+   */
   public getHash(): string {
     return this.attestation.claimHash
   }
 
+  /**
+   * @description Builds a presentation. A presentation is a custom view of the [[AttestedClaim]], in which the claimer controls what information should be showed.
+   * @param excludedClaimProperties An array of claim properties to **exclude**.
+   * @param excludeIdentity Whether the claimer's identity should be **excluded** from the presentation.
```

Comment:
I think mentioning the excludeIdentity is set to false default might make it easier to read or a boolean. 

It took me two read-throughs to understand what the 'true' in the function was doing.

Reply:
done